addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/ResourceIdentificationUtils.java

```diff
@@ -31,12 +37,11 @@ private ResourceIdentificationUtils() {}
             Pattern.compile("(?:\\.js|\\.jsx)\\z", Pattern.CASE_INSENSITIVE);
 
     /**
-     * Returns whether or not the given {@link HttpMessage} has a font content type in its response
-     * or request headers.
```

Comment:
“content type in its response or request URL path"

Everywhere the content type and path are checked.